frontend/gql/queries.js

```diff
@@ -0,0 +1,18 @@
+import { gql } from '@apollo/client';
+
+ export const FIND_USER = gql`
+   query GetUser($uid: String!) {
```

Comment:
The variable is called FIND_USER and the query is GetUser 😆 I know this is a nitpick but as a general rule of thumb, pick one and stick to it, it can kill a lot of confusion in the long run.